wai-handler-hal/wai-handler-hal.cabal

```
@@ -38,16 +38,16 @@ common opts
 common deps
   build-depends:
     , base                  >=4.12    && <4.15
-    , base64-bytestring     ^>=1.1.0.0
+    , base64-bytestring     ^>=1.0.0.0
     , bytestring            ^>=0.10.8
-    , case-insensitive      ^>=1.2.1.0
+    , case-insensitive      ^>=1.2.0.0
     , hal                   ^>=0.4.7
     , http-types            ^>=0.12.3
-    , network               ^>=3.1.1.1
+    , network               ^>=2.8.0.0
```

Your lower bound will exclude `network ^>= 3.0` and `network ^>= 3.1`. How about this?
```suggestion
    , network               >=2.8.0.0 && <3.2
```

---

I have read the changelogs for `network` and am satisfied that these bounds are safe.

---

Yes, that was my bad. Thanks for catching it.